spec/report_spec.rb

```
@@ -512,14 +512,15 @@ def gloops
     }
   end
 
-  it "marks the top-most stacktrace line as inProject if necessary" do
+  it "marks the top-most non-bugsnag stacktrace line as inProject if necessary" do
     Bugsnag.configuration.project_root = File.expand_path File.dirname(__FILE__)
     Bugsnag.notify(BugsnagTestException.new("It crashed"))
```

We should raise this exception so it has a populated stacktrace